docs/reference/search/request/sort.asciidoc

```diff
@@ -26,6 +26,13 @@ field name for `_score` to sort by score.
 The sort values for each document returned are also returned as part of
 the response.
 
+==== Sort Order
+
+The `order` option can have the following values:
+[horizontal]
+`asc`:: (Default) Sort in ascending order
```

Comment:
Actually, the default when sorting on `_score` (the default sort) is descending, not ascending.  For any other values it is ascending